playbooks/openstack/configuration.md

```diff
@@ -1075,3 +1076,68 @@ $ ansible-playbook --user openshift \
   -i inventory \
   openshift-ansible/playbooks/openstack/openshift-cluster/install.yml
 ```
+
+## Using CRI-O
```

Comment:
Its recommended to use `node-config-<role>-crio` node config for CRI-O nodes, it would be nice to have this mentioned in this section